vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/graal/AddDataFromDlgpFile.java

```diff
@@ -0,0 +1,108 @@
+package org.semanticweb.vlog4j.examples.graal;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.semanticweb.vlog4j.core.reasoner.Reasoner;
+import org.semanticweb.vlog4j.core.reasoner.exceptions.EdbIdbSeparationException;
+import org.semanticweb.vlog4j.core.reasoner.exceptions.IncompatiblePredicateArityException;
+import org.semanticweb.vlog4j.core.reasoner.exceptions.ReasonerStateException;
+import org.semanticweb.vlog4j.examples.ExamplesUtils;
+import org.semanticweb.vlog4j.graal.GraalConjunctiveQueryToRule;
+import org.semanticweb.vlog4j.graal.GraalToVLog4JModelConverter;
+
+import fr.lirmm.graphik.graal.api.core.Atom;
+import fr.lirmm.graphik.graal.api.core.ConjunctiveQuery;
+import fr.lirmm.graphik.graal.api.core.Rule;
+import fr.lirmm.graphik.graal.io.dlp.DlgpParser;
+
+/**
+ * This example shows how facts can be imported from files in the DLGP/DLP
```

Comment:
Please add a link to http://graphik-team.github.io/graal/doc/dlgp when referencing DLGP format.

Reply:
Added hyperlink to the documentation.